vignettes/art-contrasts.Rmd

```diff
@@ -196,26 +203,22 @@ subtracted out the effect of `C - D`. We can see this if we take the ART estimat
 ART estimate for `A,C - A,D` (`125.12`) here, we can get approximate an estimate of the difference (`-123.13 + 125.12 == 1.99`)
 that is consistent with the expected 0 (given the SE here).
 
-If we first combine the factors before aligning-and-ranking, we can get an estimate of these effects:
+The ART-C procedure however, was developed to align and rank data specifically for contrasts involving levels from any number of factors.
 
 ```{r}
-#DO NOT DO THIS WITHOUT READING THE NOTE BELOW
-df$X = with(df, X1:X2)
-m.art.12 = art(Y ~ X, data=df)
-contrast(emmeans(artlm(m.art.12, "X"), ~ X), method="pairwise")
+art.con(m.art, "X1:X2")
 ```
 
-Note that the above table is not in the same order as the previous ones, but does have similar t values to the corresponding effects
-from the linear model. __However, it is not clear this will generalize.__ In this case there aren't any other factors except these two, 
-so this is equivalent to just ranking the response before fitting the model, and in the general case it can incorrectly estimate 
-interactions (see issues with using the plain-old rank transform in the ART literature, specifically Higgins' papers). Thus, I 
-can't recommended this approach in the general case (or at least, not without more investigation).
+Like the linear model, art.con  correctly estimates the difference between `A,C - A,D` as `~0`. In fact, its results agree with the linear model for all contrasts conducted. (Note that the art.con and linear model results appear in a different order; specifically A C - A D appears second for the linear model and first for art.con)
+
+The syntax used above is consistent with *term* syntax used by artlm. art.con also accepts the *formula* syntax accepted by emmeans. We can conduct the same contrasts as above using the following syntax:
+```{r}
+art.con(m.art, ~X1*X2)
+```
 
 ### Tests of _differences of differences_ in interactions
 
-While tests of differences of combinations of levels between factors have issues in ART, we can test _differences of differences_; e.g., 
-for the interaction `X1:X2`, we might ask, is the difference `A - B` different when `X2 = C` compared to when `X2 = D`? We
-can test this using the `interaction` argument to the `contrast` function in the `emmeans` package.
+Before ART-C was created and added to this package, the only way to conduct multifactor contrasts within the ART paradigm was to test _differences of differences_; e.g., for the interaction `X1:X2`, we might ask, is the difference `A - B` different when `X2 = C` compared to when `X2 = D`?. We can still test this using the `interaction` argument to art.con. Note that when the `interaction` argument is supplied to art.con, differences of differences are tested on data that has been aligned-and-ranked using the **original** ART method (i.e., the data is **not** aligned-and-ranked using the ART-C method).
```

Comment:
I would start with the simple art.con example and interpret it, then move the old (manual) approach after the interpretation text down where the phia stuff is.